lib/sti_friendly.rb

```diff
@@ -0,0 +1,29 @@
+require 'sti_friendly/version'
+
+module StiFriendly
+  # Wrapper around becomes that also changes the instance's sti column value.
+  # This is especially useful if you want to persist the changed class in your
+  # database.
+  def becomes!(klass)
+    became = becomes(klass)
+    sti_type = nil
+    unless klass.descends_from_active_record?
```

Comment:
Favor modifier `unless` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.